Core/wbDefinitionsTES5.pas

```diff
@@ -10760,7 +10760,7 @@ procedure DefineTES5m;
     wbSPCT,
     wbSPLOs,
     wbDEST,
-    wbFormIDCk(WNAM, 'Worn Armor', [ARMO], False, cpNormal, False),
+    wbFormIDCk(WNAM, 'Skin', [ARMO], False, cpNormal, False),
```

Comment:
Technically, it defines the default armor (because body mesh is an armor) occupying slot 32 when character either wears no clothes or clothing pieces that do not have their own body meshes. Suggestion: "Skin (Default Body Armor)"

Reply:
It's labeled Skin in the CK.  and you only use this on non-human creatures with the Skin ARMO that is the actual creature 
.nif, for example Draugr.